shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/antlr4/imports/Keyword.g4

```diff
@@ -82,7 +82,23 @@ FROM
 SHARDING
     : S H A R D I N G
     ;
-   
+
+READ_WRITE_SPLITTING
```

Comment:
```yaml
rules:
- !READWRITE_SPLITTING
```
shows us `readwrite_spliting`, is it better to keep the same keyword here?

Reply:
yes, I will update the issue latter.